static/src/javascripts/projects/common/modules/experiments/tests/sign-in-engagement-banner-display.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+export const signInEngagementBannerDisplay: ABTest = {
+    id: 'SignInEngagementBannerDisplay',
+    start: '2018-04-26',
+    expiry: '2018-05-26',
```

Comment:
This looks like a Saturday, might be better to choose a weekday?

Reply:
ah yes, I just added +30 days without looking!

Reply:
Ha! Might not be a big deal. But I like someone to be in the office when a test expires just in case there's some unexpected consequence